.github/workflows/test.yml

```diff
@@ -10,16 +10,16 @@ jobs:
       fail-fast: false
       matrix:
         node: [18, 20]
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-latest, macos-latest, macos-14, windows-latest]
```

Comment:
the link you provided (https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) says `macos-latest` is equivalent `macos-14`, so I don't think we need both of these as it seems a bit redundant